### ISSUE TITLE
Jazzy Ignores Xcodebuild Arguments.

### ISSUE THREAD

**Issue by wmcbain** [realm/jazzy#674]
**Environment Details**

* macOS 10.12.1
* Jazzy 0.7.2
* Xcode 8.1 8B62

**Configuration File**
```yaml
output: Docs
clean: true
sdk: macosx
xcodebuild_arguments: [clean,build,-project,ZXFoundation.xcodeproj,-scheme,ZXFoundation-macOS]
source_directory: Sources
author: ZX Ventures
author_url: https://zxventures.com
module: ZXFoundation
module_version: 0.23.0
copyright: 2016 [ZX Ventures LLC](https://zxventures.com)
readme: README.md
podspec: ZXFoundation.podspec
github_url: https://github.com/ZXVentures/ZXFoundation.git
min_acl: internal
theme: fullwidth
```

**Expected**
* Jazzy uses defined Xcode arguments to select proper Xcodeproj and target.

**Actual**
* Jazzy ignores defined arguments and fails when it tries to build the documentation.

**Output**

* Command line:
```
$  jazzy
Using config file /Users/myuser/Sites/ZXVentures/ZXFoundation/.jazzy.yaml
Analyzing dependencies
Fetching podspec for `ZXFoundation` from `/Users/myuser/Sites/ZXVentures/ZXFoundation`
Downloading dependencies
Installing ZXFoundation (0.23.0)
Generating Pods project
Skipping User Project Integration
Pod installation complete! There is 1 dependency from the Podfile and 1 total pod installed.
Running xcodebuild
Could not parse compiler arguments from `xcodebuild` output.
Please confirm that `xcodebuild` is building a Swift module.
Saved `xcodebuild` log file: /var/folders/4v/90mgrh551jq_99g7nr_1tgmh0000gn/T/xcodebuild-D3DB4B54-6930-489D-B646-47752FE41248.log
Failed to generate documentation
/Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/executable.rb:36:in `execute_command': /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/SourceKitten/bin/sourcekitten ["doc", "--module-name", "ZXFoundation", "--", "-target", "ZXFoundation-iOS"] (RuntimeError)

Running xcodebuild

Could not parse compiler arguments from `xcodebuild` output.

Please confirm that `xcodebuild` is building a Swift module.

Saved `xcodebuild` log file: /var/folders/4v/90mgrh551jq_99g7nr_1tgmh0000gn/T/xcodebuild-D3DB4B54-6930-489D-B646-47752FE41248.log

Failed to generate documentation
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/sourcekitten.rb:147:in `run_sourcekitten'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/podspec_documenter.rb:25:in `block (3 levels) in sourcekitten_output'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/podspec_documenter.rb:24:in `map'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/podspec_documenter.rb:24:in `block (2 levels) in sourcekitten_output'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/podspec_documenter.rb:19:in `chdir'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/podspec_documenter.rb:19:in `block in sourcekitten_output'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/cocoapods-1.1.1/lib/cocoapods/config.rb:41:in `with_changes'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/podspec_documenter.rb:15:in `sourcekitten_output'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/lib/jazzy/doc_builder.rb:60:in `build'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/gems/jazzy-0.7.2/bin/jazzy:15:in `<top (required)>'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/bin/jazzy:23:in `load'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/bin/jazzy:23:in `<main>'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/myuser/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `<main>'
```

* Log file:

```
Build settings from command line:
    CODE_SIGN_IDENTITY =
    CODE_SIGNING_REQUIRED = NO

=== CLEAN TARGET ZXFoundation-iOS OF PROJECT Pods WITH THE DEFAULT CONFIGURATION (Release) ===

Check dependencies
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

** CLEAN FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
=== BUILD TARGET ZXFoundation-iOS OF PROJECT Pods WITH THE DEFAULT CONFIGURATION (Release) ===

Check dependencies
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
```

**Comment by orta** [comment]
The podspec attribute is overwriting your build settings, might be worth either removing that - or making sure you're on the latest CocoaPods - looks like it isn't building due to a swift version not being set

**Comment by wmcbain** [comment]
Hey as other issues have stated setting `swift_version` causes the following with Xcode 8:

```
xcrun: error: unable to find utility "xcrun", not a developer tool or in PATH
```

However removing the podspec key did help and after playing around with Issue 677 I realized my build arguments were wrong as I was targeting a scheme in a workspace rather than an xcodeproj. My updated configuration looks as follows and this builds documentation appropriately:

```yaml
output: Docs/Site
sdk: ios
xcodebuild_arguments: [clean,build,-workspace,ZXFoundation.xcworkspace,-scheme,ZXFoundation-iOS]
source_directory: ./
author: ZX Ventures
author_url: https://zxventures.com
module: ZXFoundation
module_version: 0.23.0
copyright: 2016 [ZX Ventures LLC](https://zxventures.com)
readme: README.md
github_url: https://github.com/ZXVentures/ZXFoundation.git
min_acl: internal
theme: fullwidth
```

**tl;dr** In my `xcodebuild` arguments I needed to target a workspace and a scheme, rather than a project and a scheme.

Closing as this is solved.
